### PR TITLE
Sell exit bridge

### PR DESCRIPTION
**Pull request by eezcjkr**

### CODE REVIEW

File: contracts/ParsecBridge.sol [hunk 1]
```diff
@@ -547,6 +547,72 @@ contract ParsecBridge is Ownable {
...
+
+    // validate proof
+    bytes32 txHash;
+    bytes memory txData;
+    (, txHash, txData) = TxLib.validateProof(64, _proof);
+    // parse tx and use data
+    TxLib.Tx memory txn = TxLib.parseTx(txData);
+    TxLib.Output memory out = txn.outs[_oindex];
+    (uint256 buyPrice, bytes32 utxoIdSigned, address signer) = unpackSignedData(signedData);
+    // address sender = ecrecover(TxLib.getSigHash(txData), txn.ins[0].v, txn.ins[0].r, txn.ins[0].s);
```
Comment (johannbarbie) [inline contracts/ParsecBridge.sol]: remove

File: contracts/ParsecBridge.sol [hunk 2]
```diff
@@ -547,6 +547,72 @@ contract ParsecBridge is Ownable {
...
+
+    require(out.owner == address(this), "Funds were not sent to bridge");
+    require(ecrecover(TxLib.getSigHash(txData), txn.ins[0].v, txn.ins[0].r, txn.ins[0].s) == signer, "Exit was not signed by owner");
+
+    uint256 exitableAt = Math.max256(periods[_proof[0]].timestamp + (2 * exitDuration), block.timestamp + exitDuration);
+    // bytes32 utxoIdProof = bytes32((_oindex << 120) | uint120(txHash));
```
Comment (johannbarbie) [inline contracts/ParsecBridge.sol]: remove

File: contracts/ParsecBridge.sol [hunk 3]
```diff
@@ -547,6 +547,72 @@ contract ParsecBridge is Ownable {
...
+
+    require(bytes32((_oindex << 120) | uint120(txHash)) == utxoIdSigned, "The signed utxoid does not match the one in the proof");
+
+    uint256 priority = (exitableAt << 128) | uint128(utxoIdSigned);
+    require(out.value > 0);
+    require(exits[utxoIdSigned].amount == 0);
+    if (isNft(out.color)) {
+      nftExits[out.color].push(NftExit({ utxoId: utxoIdSigned, exitableAt: exitableAt }));
```
Comment (johannbarbie) [inline contracts/ParsecBridge.sol]: i don't think we can sell NFTs this way, `tokens[out.color].addr.transferFrom(msg.sender, signer, buyPrice);` will not work. or are you thinking of a kind of NFT swap?

Comment (eezcjkr) [reply contracts/ParsecBridge.sol]: You are right, I copy pasted the standard exit as my starting point and left this in for some reason. Will remove.

**Pull request merged**